packages/service-library/src/servicelib/background_task.py

```diff
@@ -64,3 +64,22 @@ async def stop_periodic_task(
         asyncio_task.cancel()
         with log_catch(logger, reraise=False):
             await asyncio.wait((asyncio_task,), timeout=timeout)
+
+
+@contextlib.asynccontextmanager
+async def periodic_task(
```

Comment:
MINOR: something in the name highlight that this is a context manager? e.g. ``periodic_task_context`` or ``periodic_task_lifespam``?

Reply:
I am actually pretty happy with this name, and pylance nicely show it is a context manager.